.travis.yml

```diff
@@ -1,50 +0,0 @@
-language: scala
-
-scala:
-  - 2.11.12
-  - 2.12.10
-  - 2.13.1
-
-jdk:
-  - openjdk8
-
-dist: xenial
-sudo: false
-
-cache:
-  directories:
-    - $HOME/.ivy2/cache
-    - $HOME/.sbt/boot
-    - $HOME/.coursier
-
-deploy:
-  - provider: releases
-    api_key:
-      secure: NaItKK8ZQUbMJUDqtSAgfE0idFF2dDb27yGvWZYT6K9E2DWaSjY4GCQM6uhg2TkaiyMMcsy+19nls8iHm/tfgrU6yLRhc3TpEDL6MTi+Db1LsXQbu6gpOhBEl5K0352lIi0RqEeVWpDiPT+kErg2U/Njq0TSkuAnGAUNYMgeRuc0cGgR5np7dYOSJpW9TO4lV9NDmkALl8EnVx94TlXlo94rsht1Io9Gse7gnRajXdyyk+iBAckATodOeBN42Zg/6ymUwhzeaEn6imVFiUSW/4WYYZyoNEvrevBw6+tqdg9MM10CkRcYe1cnos0rsMm1/05QE3IszhtOZ1lSay8P0xUS615Kmgo751Kf6s5oBeylx5MIQAroS4VvHo4XrojiAT8t0b4S/6bsfIGSax3wsw37P4SegACo8DUEImmHCWL+lCWNjcY3+AOfoaepJJk5guxzQC9nB3Q9OXfbyc67JnJJfqyMDmyHq52sGixze9NDsc+e59nA7kwJE+GuwRf8NjBrFgqFx76eXDPWKd+9EZB6w6/xTfzDVoQ8YmWAhbCfJ8TklEVh1kmX6o5PTcBGDTzrWfb8EA9LyV5g426CA4cqP2iQLMz+BocBNEHg4hpVPa9krF5hd27ReOjwIMju1MJnEmN5OmI9E5YwgZDdaIkOKaeb4zY4zUScWiEcbXY=
```

Comment:
where was this API key coming from? Since we don't need it anymore, can we delete it?

Reply:
It's a github api key generated by travis. There isn't really anything to delete besides revoking access to travis once everything is moved to github actions.